components/shared/normalizeChars.js

```diff
@@ -1,14 +1,15 @@
 const normalizeChars = value =>
   value.replace(
-    /([àáâãäå])|([çčć])|([èéêë])|([ìíîï])|([òóôõöø])|([ùúûü])|([-_])/g,
-    (str, a, c, e, i, o, u, hifen) => {
+    /([àáâãäå])|([çčć])|([èéêë])|([ìíîï])|([òóôõöø])|([ùúûü])|([-_])|(['])/g,
+    (str, a, c, e, i, o, u, hifen, otherToRemove) => {
```

Comment:
doubt: What reason of this args name?
folowing the criterion of other arguments, shouldnt it be `quote`?